From f80d4ba81964cff319815771fd520c1df9838cc6 Mon Sep 17 00:00:00 2001
From: Olivier Humbert <trebmuh@users.noreply.github.com>
Date: Tue, 22 Nov 2016 00:20:32 +0100
Subject: [PATCH 1091/1936] Create TODO-release

Hope that could help in some way to start the ball rolling :)
---
 TODO-release | 6 ++++++
 1 file changed, 6 insertions(+)
 create mode 100644 TODO-release

diff --git a/TODO-release b/TODO-release
new file mode 100644
index 00000000..d25a1258
--- /dev/null
+++ b/TODO-release
@@ -0,0 +1,6 @@
+This is a list of things to do when releasing a new version.
+
+- Update version number in CMakeLists.txt
+- Update date of the new version in HISTORY.txt
+- Update Copyright (mainly YEAR) info in src/UI/MasterUI.fl (note this is mentionned 2 times on line 5 and line 872)
+- Check and update (if needed) the manpage file : doc/zynaddsubfx.1.txt
-- 
2.47.0

